### DISCUSSION TITLE
希望<文件命名格式>能增加获取up主uid和up主昵称

### DISCUSSION THREAD

**Discussion by hunterHungry** [the1812/Bilibili-Evolved#4658] [功能建议 / Ideas]
以个人为例，常用的的格式是 `Title - uid . up主昵称`，希望 文件命名格式里能增加这两项。

另外<批量命名格式>希望能增加先导零，例如，`01 - Title` 这样。

**Comment by Geeyun-JY3** [comment]
部分重复：#2134

**Comment by the1812** [comment]
先导零一直都支持的, 前提是批量的数量大于 10
